dev/charts/charts.html

```diff
@@ -0,0 +1,1359 @@
+      </template>
+    </dom-bind>
+
+    <vaadin-chart type="bar" title="Historic World Population by Region" subtitle='Source: <a href="https://en.wikipedia.org/wiki/World_population">Wikipedia.org</a>' categories='["Africa", "America", "Asia", "Europe", "Oceania"]' additional-options='{
+      "yAxis": {
+        "min": 0,
+        "title": {
+          "text": "Population (millions)",
+          "align": "high"
+        }
+      },
+      "tooltip": {
+        "valueSuffix": " millions"
+      },
+      "plotOptions": {
+        "bar": {
+          "dataLabels": {
+            "enabled": true
+          }
+        }
+      }
+    }'>
+      <vaadin-chart-series title="Year 1800" values="[107, 31, 635, 203, 2]">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Year 1900" values="[133, 156, 947, 408, 6]">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Year 2000" values="[814, 841, 3714, 727, 31]">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Year 2016" values="[1216, 1001, 4436, 738, 40]">
+      </vaadin-chart-series>
+    </vaadin-chart>
+
+    <vaadin-chart type="boxplot" title="Box Plot Example" categories='["1", "2", "3", "4", "5"]' no-legend tooltip additional-options='{
+      "xAxis": {
+        "title": {
+          "text": "Experiment No."
+        }
+      },
+      "yAxis": {
+        "title": {
+          "text": "Observations"
+        },
+        "plotLines": [{
+          "value": 932,
+          "label": {
+            "text": "Theoretical mean: 932",
+            "align": "center"
+          }
+        }]
+      }
+    }'>
+      <vaadin-chart-series title="Observations" values="[
+      [760, 801, 848, 895, 965],
+      [733, 853, 939, 980, 1080],
+      [714, 762, 817, 870, 918],
+      [724, 802, 806, 871, 950],
+      [834, 836, 864, 882, 910]
+    ]" additional-options='{
+      "tooltip": {
+        "headerFormat": "<em>Experiment No {point.key}</em><br/>"
+      }
+    }'>
+      </vaadin-chart-series>
+      <vaadin-chart-series type="scatter" title="Outlier" values="[[0, 644], [4, 718], [4, 951], [4, 969]]" additional-options='{
+      "tooltip": {
+        "pointFormat": "Observation: {point.y}"
+      }
+    }'>
+      </vaadin-chart-series>
+    </vaadin-chart>
+
+    <vaadin-chart type="bubble" title="Vaadin Charts Bubbles" tooltip>
+      <vaadin-chart-series title="All bubbles shown" values="[[9, 81, 13], [98, 5, 39], [51, 50, 23], [41, 22, -36], [58, 24, -30], [78, 37, -16], [55, 56, 3], [18, 45, 20], [42, 44, -22], [3, 52, 9], [31, 18, 47], [79, 91, 13], [93, 23, -27], [44, 83, -28]]" additional-options='{"maxSize": 120, "minSize": 3}'>
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Negative bubbles hidden" values="[[13, 30, 10], [23, 20, -10], [23, 40, 10]]" additional-options='{"displayNegative": false}'>
+      </vaadin-chart-series>
+    </vaadin-chart>
+
+    <dom-module id="bullet-theme" theme-for="vaadin-chart">
```

Comment:
Consider using `registerStyles` instead of `dom-module` to not import the legacy adapter.